src/nginxconfig/templates/global_sections/performance.vue

```diff
@@ -154,6 +158,10 @@ THE SOFTWARE.
     import PrettyCheck from '../inputs/checkbox';
 
     const defaults = {
+        disableHtmlCaching: {
+            default: true,
```

Comment:
```suggestion
            default: false,
```
To ensure backwards compatibility in the tool, let's set this to false by default